Trimble.Modus.Components/Controls/Button/TMButton.cs

```diff
@@ -0,0 +1,407 @@
+﻿
+using Microsoft.Maui.Controls.Shapes;
+
+
+namespace Trimble.Modus.Components
+{
+
+    public class TMButton : ContentView
```

Comment:
Can you add "Clicked" property for button

Reply:
Added Clicked Property